app/javascript/src/components/Dashboard/Contacts/utils.jsx

```diff
@@ -64,8 +77,11 @@ export const buildContactColumnData = (
     dataIndex: "dropdown_menu",
     key: "dropdown_menu",
     title: "",
-
     render: (item, { id }) =>
```

Comment:
If a prop is unused then use `_`. Refer: https://stackoverflow.com/questions/27637013/what-is-the-meaning-of-an-underscore-in-javascript-function-parameter
```suggestion
render: (_, { id }) =>
```

Also a question: Didn't Eslint complain about the unused variable `item`? From my understanding Eslint will complain. Make sure to lemme know what's shown when you run eslint over this file.

Reply:
I will fix this.
Also `eslint` didn't pop up any warnings/errors during the commit phase. For reconfirming I tried to recreate the same issue and it worked well.<img width="1406" alt="Screenshot 2022-09-22 at 2 46 21 PM" src="https://user-images.githubusercontent.com/57584818/191712685-50916a09-9e31-4e37-8329-a5c7422875c3.png">

Reply:
> I will fix this. Also `eslint` didn't pop up any warnings/errors during the commit phase. For reconfirming I tried to recreate the same issue and it worked well.<img alt="Screenshot 2022-09-22 at 2 46 21 PM" width="1406" src="https://user-images.githubusercontent.com/57584818/191712685-50916a09-9e31-4e37-8329-a5c7422875c3.png">

But the same is working if I add some unused state variable<img width="1440" alt="Screenshot 2022-09-22 at 3 13 55 PM" src="https://user-images.githubusercontent.com/57584818/191714983-a2112aa2-41a8-4c74-806e-b83c1e9643b2.png">

Reply:
I have raised an issue for this. This will be handled once I add that rule to wheel: https://github.com/bigbinary/wheel/issues/906